sql/src/main/java/io/crate/planner/operators/FetchOrEval.java

```diff
@@ -430,6 +430,17 @@ private ExecutionPlan planWithEvalProjection(PlannerContext plannerContext, Exec
         return executionPlan;
     }
 
+    @Override
+    public LogicalPlan tryOptimize(@Nullable LogicalPlan pushDown) {
+        if (pushDown instanceof Order) {
+            LogicalPlan newPlan = source.tryOptimize(pushDown);
```

Comment:
I wonder if this is safe if we've something like

```
Order  f(x)
   |
FetchOrEval  f(x)
   |
...
   |
Collect x
```

(FetchOrEval having a scalar and the order by depending on the execution of that scalar)